### ISSUE TITLE
Fingerprints break for new version of `scikit-fingerprints`

### ISSUE THREAD

**Issue by AVHopp** [emdgroup/baybe#446]
With the newest version 1.12.0 of `scikit-fingerprints`, our code breaks. This can be seen when e.g. running the `Basics/campaign` example which fails when having package version 1.12.0 but works for 1.11.0.

```bash
m290244@deda1x5085:~/baybe> python examples/Basics/campaign.py 
Traceback (most recent call last):
  File "/home-cdo/m290244/baybe/examples/Basics/campaign.py", line 60, in <module>
    searchspace = SearchSpace.from_product(parameters=parameters)
  File "/home-cdo/m290244/baybe/baybe/searchspace/core.py", line 145, in from_product
    discrete = SubspaceDiscrete.from_product(
  File "/home-cdo/m290244/baybe/baybe/searchspace/discrete.py", line 253, in from_product
    return SubspaceDiscrete(
  File "<attrs generated init baybe.searchspace.discrete.SubspaceDiscrete>", line 14, in __init__
    _setattr('comp_rep', __attr_factory_comp_rep(self))
  File "/home-cdo/m290244/baybe/baybe/searchspace/discrete.py", line 166, in _default_comp_rep
    comp_rep = self.transform(self.exp_rep)
  File "/home-cdo/m290244/baybe/baybe/searchspace/discrete.py", line 688, in transform
    comp_df = param.transform(df[param.name])
  File "/home-cdo/m290244/baybe/baybe/parameters/base.py", line 142, in transform
    right=self.comp_df,
  File "/home-cdo/m290244/miniforge3/envs/baybe310/lib/python3.10/functools.py", line 981, in __get__
    val = self.func(instance)
  File "/home-cdo/m290244/baybe/baybe/parameters/substance.py", line 135, in comp_df
    comp_df = chemistry.smiles_to_fingerprint_features(
  File "/home-cdo/m290244/baybe/baybe/utils/chemistry.py", line 152, in smiles_to_fingerprint_features
    col_names = [
  File "/home-cdo/m290244/baybe/baybe/utils/chemistry.py", line 153, in <listcomp>
    prefix + name + f.split("fingerprint")[1]
IndexError: list index out of range
```

Explicitly pinging @Hrovatin here as she created this feature. Will create a hotfix PR pinning the version for now.

**Comment by fabianliebig** [comment]
I could reproduce that error on a fresh installed environment as well.

**Comment by Scienfitz** [comment]
Note that I implemented the passthrough of explicit descriptor names in `.get_feature_names_out()` into skfp for mordred and rdkit2ddescriptors which should be part of the new release
Not sure if related to the issue but was a point of discussion before

**Comment by AVHopp** [comment]
Resolved by #447 .
